### ISSUE TITLE
Phone app: transfers do not work

### ISSUE THREAD

**Issue by MiriamNZ** [actualbudget/releases#205]
You choose a transfer account from the payee list. 
Save the transaction. 
![image](https://user-images.githubusercontent.com/6721002/153956034-4c2a064a-e5b3-4b48-b5b0-b353b0b13561.jpeg)
The transactions lists as needing a category. 
The destination account does not get the mirroring transaction.

So now account balances are incorrect (one account is debited but the other is not credited).

**Comment by Kidglove57** [comment]
I have not been able to reproduce this on iOS. Might it relate to the issue raised on Slack where a Payee had been imported to Actual with the words Transfer in it. That Payee then having been selected in mistake for a real transfer. When I correctly set the destination transfer account, no category is requested.

I believe the person who posted on Slack had to go through and tidy up their payees list to avoid the ongoing issue.

**Comment by MiriamNZ** [comment]
When I look at a proper transfer (done on the computer), the payee says "Transfer from Bills".

When I make a transaction, which is a transfer,  I click on Payee, and what comes up is a list of Payees, and at the bottom a list of Transfer accounts. Each account in my account list appears, but with a colon in the name "Transfer: Bills".
There is no other way I can see to do a transfer on the app.

Do I just not understand how to do a transfer on the app? How do you do it and what do you see?
I will try and see if I can delete all those "Transfer:" items from my payee list.

Cheers
Miriam

**Comment by MiriamNZ** [comment]
OK, so I worked out how to get at my pay list (go to accounts, find a transaction, click on the payee, and the option to manage payees appears).
I deleted all the "transfer: ..." items.
I had to resync to get phone app and computer on the same page.

On the phone, the payee list still has the colon in the name in the ist "Transfer: ..." but now they work as a transfer, no category required. When I choose the item, it appears I the Payee field without the colon, so looking just like it does in the computer app.

Thanks for your help on this. Problem solved.
Cheers
Miriam
